distribution/lib/Standard/AWS/0.0.0-dev/src/Internal/Auth.enso

```diff
@@ -0,0 +1,13 @@
+from Standard.Base import all
+
+import project.AWS_Credential.AWS_Credential
+
+polyglot java import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider
+
+## PRIVATE
+   Create the Java credentialsProvider object.
+create_provider : (AWS_Credential | Nothing) -> ProfileCredentialsProvider
+create_provider credentials:(AWS_Credential | Nothing) = case credentials of
+    AWS_Credential.Profile p -> ProfileCredentialsProvider.builder.profileName(p).build
+    AWS_Credential.Key k s -> ProfileCredentialsProvider.builder.accessKeyId(k).secretAccessKey(s).build
```

Comment:
wait this Java-line syntax works?! it must be accidentally due to some abuse of `()` precedence. I guess we may consider doing it if we want to, but I'm not sure - it can confuse readers - creating the impression that the general Java syntax works and ppl will be confused why `foo.bar(x)` worked but `foo.bar(x, y)` did not.

I think it will be better if we stick to Enso syntax:
```suggestion
    AWS_Credential.Profile p -> ProfileCredentialsProvider.builder.profileName p . build
    AWS_Credential.Key k s -> ProfileCredentialsProvider.builder.accessKeyId k . secretAccessKey s . build
```